docker/Dockerfile

```
@@ -6,7 +6,7 @@ ENV ANSIBLE_RETRY_FILES_ENABLED=false
 ENV ANSIBLE_STDOUT_CALLBACK=debug
 
 RUN apk add --no-cache --update bash git mysql-client openssh-client postgresql rsync
```

```suggestion
RUN apk add --no-cache --update bash git mysql-client openssh-client postgresql rsync sshpass
```
\+ retirer `sshpass` de la ligne d'après (je ne peux pas faire de suggestion sur cette ligne via GitHub)

La 2ème ligne de `apk add` ne sert que pour des dépendances qu'on installe juste le temps de compiler l'image, puis on les supprime juste après.
Pour les dépendances pérennes c'est dans la 1ère.

---

Merci je n'avais outres fait attention